src/Bridges/Constraint/quad_to_soc.jl

```diff
@@ -153,7 +153,8 @@ function MOI.supports_constraint(::Type{QuadtoSOCBridge{T}},
                                                 MOI.GreaterThan{T}}}) where T
     return true
 end
-function added_constraint_types(::Type{QuadtoSOCBridge{T}}) where T
+MOIB.added_constrained_variable_types(::Type{<:QuadtoSOCBridge}) = Tuple{DataType}[]
```

Comment:
Why is it necessary to add these to every bridge? Shouldn't there be a generic fallback?